packages/components/src/custom-select-control-v2/legacy-component/test/index.tsx

```diff
@@ -286,7 +286,7 @@ describe.each( [
 			expect.objectContaining( {
 				inputValue: '',
 				isOpen: false,
-				selectedItem: { key: 'violets', name: 'violets' },
+				selectedItem: { key: 'flower1', name: 'violets' },
```

Comment:
The fact it had the same `key` and `value` looked like a bug caused by the previous logic [here](https://github.com/WordPress/gutenberg/pull/61272/files#diff-098732f0ef5f857dc8edaffd70c0cbd3b1388defa30a6689ba197b2cd70e3a9aL48-L51).